stage0/run.go

```diff
@@ -466,5 +468,5 @@ func launchMetadataService(debug bool) error {
 		Stderr: os.Stderr,
 	}
 
-	return cmd.Start()
+	return cmd.Run()
```

Comment:
@jonboulle @philips Does it make sense to make metadata service optional in the spec? This would (a) lower the bar to making a compliant ACE; (b) not require users to start metadata service for pods that don't use them.